website/docs/ConfigurationFile.md

```diff
@@ -128,9 +128,9 @@ export const config = {
         // maxInstances can get overwritten per capability. So if you have an in house Selenium
         // grid with only 5 firefox instance available you can make sure that not more than
         // 5 instance gets started at a time.
-        maxInstances: 5,
+        'wdio:maxInstances': 5,
         browserName: 'firefox',
-        specs: [
+        'wdio:specs': [
```

Comment:
Can we make this change across all translations as well?

Reply:
Updated